Dockerfile

```diff
@@ -18,11 +19,12 @@ RUN \
     groupadd --gid 10001 app && \
     useradd --uid 10001 --gid 10001 --home /app --create-home app && \
     apt-get -q update && \
-    apt-get -q install -y default-libmysqlclient-dev libssl-dev ca-certificates libcurl4 && \
-    rm -rf /var/lib/apt/lists
+    apt-get -q install -y --no-install-recommends default-libmysqlclient-dev libssl-dev ca-certificates libcurl4 python3.7 python3-pip && \
```

Comment:
@erkolson - do you need pip as well? I added it here, but can remove if you don't think it's necessary. I assumed you might since we still need to install the google SDK, but if you have another approach there, let me know and I'll remove `python3-pip` so we're not adding stuff we don't need.

Reply:
We came up with a slightly different plan at the Monday meeting, don't think we need python anymore.  Converting the tests to rust will take some time, the current tests will need python 2.7 and we don't want to put that in the rust app image.  The TTL purge script can be rewritten in rust quickly and I believe that is in progress.  To facilitate the automated tests, we will pull the old python app image to execute tests, the containers will be deleted immediately after tests pass.  This will keep the rust app image cleaner and smaller.

Reply:
That's what I get for being on vacation for the Monday meeting :) I'll close this one out then. @pjenvey - who's handling the rewrite for the purge_ttl script?

Reply:
Sorry for the wasted time/effort!

Reply:
no worries :)